package.json

```diff
@@ -4,27 +4,46 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start-dev": "nodemon ./server/app.js --exec babel-node"
   },
   "author": "Andela Simulations Programme",
   "license": "MIT",
   "dependencies": {
-    "body-parser": "^1.18.3",
+    "@babel/core": "^7.4.5",
+    "@babel/node": "^7.4.5",
+    "@babel/preset-env": "^7.4.5",
+    "@babel/register": "^7.4.4",
+    "body-parser": "^1.19.0",
```

Comment:
Why do you need this to setup sequelize

Reply:
I needed babel because i needed to import dotenv in config.js so i could use environmental variables to connect to sequelize DB

Reply:
I also needed babel to transpile the es6 code in userController.js to es5.
I needed to be certain that my sequelize setup was working well, so i created a test route that inserts a new record into users table.

Reply:
it's the `body-parser` I am referring to